### DISCUSSION TITLE
v3 bug bash

### DISCUSSION THREAD

**Discussion by jptosso** [corazawaf/coraza#377] [General]
In this thread, we will discuss requirements for the v3 bug bash

- Sponsor
- Define rewards
- Publish coverage (done using codecov)
- Prepare bases
- Define dates
- Make media arrangements (blog bosts, social media, etc)
- Assemble review team and dates

**Comment by anuraaga** [comment]
- Publish coverage to codecov

To find the bugs to bash :)
